Gruntfile.js

```diff
@@ -53,6 +53,45 @@ module.exports = exports = function(grunt) {
             package_bundling_test: {
                 src: 'test/runner.browserify.js',
                 dest: 'test/localforage.browserify.js'
+            },
+            main: {
+                files: {
+                    'dist/localforage.js': 'src/localforage.js'
+                },
+                options: {
+                    browserifyOptions: {
+                        standalone: 'localforage'
+                    },
+                    transform: ['rollupify', 'babelify'],
+                    plugin: ['bundle-collapser/plugin']
+                }
+            },
+            no_promises: {
+                files: {
+                    'dist/localforage.nopromises.js': 'src/localforage.js'
+                },
+                options: {
+                    browserifyOptions: {
+                        standalone: 'localforage'
+                    },
+                    transform: ['rollupify', 'babelify'],
+                    plugin: ['bundle-collapser/plugin'],
+                    exclude: ['lie']
+                }
+            }
+        },
+        run: {
+            derequire: {
+                cmd: './bin/derequire.js',
+                args: [
+                    'dist/localforage.js'
+                ]
+            },
+            derequire_no_promises: {
+                cmd: './bin/derequire.js',
+                args: [
+                    'dist/localforage.nopromises.js'
+                ]
```

Comment:
derequire is needed in order to have the `dist` bundle compatible with browserify itself (otherwise it complains when it sees the keyword `require`)